### ISSUE TITLE
why the Validation-accuracy drops so much when I use  mx.mod.BucketingModule() to redefine a symbol model?

### ISSUE THREAD

**Issue by janelu9** [apache/incubator-mxnet#8249]
this is a very important problem

**Comment by janelu9** [comment]
it looks that the training con't be convergent  after I use:
`    model=   mx.mod.BucketingModule(
        sym_gen             = sym_gen,
        default_bucket_key  = data_train.default_bucket_key,
        context             = [mx.gpu(int(i)) for i in range(1)])`

**Comment by janelu9** [comment]
well ,while I turn off the batch_size and learning_rate ,it looks that the Validation-accuracy runs up to the same level  as it was
